cardano-node/src/Cardano/Node/Parsers.hs

```diff
@@ -313,6 +315,14 @@ parseVrfKeyFilePath =
         <> completer (bashCompleter "file")
     )
 
+parseStartAsNonProducingNode :: Parser Bool
+parseStartAsNonProducingNode =
+  switch (
+    long "start-as-non-producing-node"
```

Comment:
`"non-producing-mode"` would be a more concise name. Users can just look at the help for more context.

Reply:
Fixed.